microsoft-graph/info.json

```diff
@@ -454,12 +454,204 @@
+                  {
+                    "tags": [],
+                    "roles": [],
+                    "osBuild": "",
+                    "verdict": "",
+                    "version": "",
+                    "riskScore": "",
+                    "osPlatform": "",
+                    "vmMetadata": "",
+                    "@odata.type": "",
+                    "mdeDeviceId": "",
+                    "rbacGroupId": "",
+                    "healthStatus": "",
+                    "ipInterfaces": [],
+                    "detailedRoles": [],
+                    "deviceDnsName": "",
+                    "lastIpAddress": "",
+                    "loggedOnUsers": [],
+                    "rbacGroupName": null,
+                    "azureAdDeviceId": null,
+                    "createdDateTime": "",
+                    "defenderAvStatus": "",
+                    "onboardingStatus": "",
+                    "firstSeenDateTime": "",
+                    "remediationStatus": "",
+                    "lastExternalIpAddress": "",
+                    "remediationStatusDetails": null
+                  }
+                ],
+                "severity": "",
+                "tenantId": "",
+                "assignedTo": "",
+                "detectorId": "",
+                "incidentId": "",
+                "systemTags": [],
+                "alertWebUrl": "",
+                "description": "",
+                "productName": "",
+                "alertPolicyId": "",
+                "determination": "",
+                "serviceSource": "",
+                "additionalData": "",
+                "classification": "",
+                "incidentWebUrl": "",
+                "createdDateTime": "",
+                "detectionSource": "",
+                "mitreTechniques": [],
+                "providerAlertId": "",
+                "actorDisplayName": "",
+                "resolvedDateTime": "",
+                "threatFamilyName": "",
+                "threatDisplayName": "",
+                "lastUpdateDateTime": "",
+                "recommendedActions": "",
+                "lastActivityDateTime": "",
+                "firstActivityDateTime": ""
+              }
+            ],
+            "@odata.context": ""
+          }
+        }
+      ],
+      "enabled": true
+    },
+    {
+      "title": "Get Security Alert",
+      "description": "Retrieves details of a specific alert from Microsoft Graph API based on the alert ID you have specified.",
+      "operation": "get_security_alert",
+      "category": "investigation",
+      "annotation": "get_security_alert",
+      "parameters": [
+        {
+          "title": "API Version",
+          "type": "select",
+          "name": "api_version",
+          "description": "Select the version of the API. If the user selects V1, this refers to the original or older version of the alerts API. If the user selects V2, this provides comprehensive information and detailed context about alerts.",
```

Comment:
If the user selects V1, this refers to the original or older version of the alerts API. If the user selects V2, this provides comprehensive information and detailed context about alerts. ---> this can be written as "v2 provides comprehensive information and detailed context about alerts, where as v1 refers to the original or older version of the alerts API."